sagui.config.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Sagui configuration object
+ */
+module.exports = {
+  /**
+   * Different application entry-points
+   * Each page is a combination of a JavaScript file and a HTML file
+   *
+   * Example: 'index' -> 'index.html' and 'index.js'
+   */
+  pages: ['index', 'source', 'target'],
```

Comment:
You can remove the `index` from the list of pages (move the adapter code to the `index.js` file as I suggested earlier) and use the [Libray feature of Sagui](https://github.com/saguijs/sagui#library) to build this guy since it is a single file.

We would just need to change the `package.json` file to point to the entry-point at `build/index.js`.

Wdyt?